app/controllers/api/base_controller.rb

```diff
@@ -20,6 +20,10 @@ def authenticate!
       render(status: :unauthorized, json: { error: "Unauthorized" })
     end
 
+    def current_provider
+      nil
```

Comment:
i think the implementation from [this PR](https://github.com/DFE-Digital/register-trainee-teachers/pull/3996/files#diff-99fc98368f0ff5548762b254189af2de831a1c6689e45225845b66a73a12fb93R30) (`@provider ||= Provider.first`) is probably sensible for the moment

Reply:
Change made @carlosmartinez 👍